### DISCUSSION TITLE
Adding the -r command to get historial data

### DISCUSSION THREAD

**Discussion by natz158** [flobz/psa_car_controller#896] [Q&A]
Afternoon everyone,

I've tried running the python3 psa-controller -f config.json -c charge_config.json -r but HA isn't reconsiging python as a command. Is there something i'm missing. HA isn't in a docker, it's running on a virtual machine on a M2 mac

**Comment by 5kat3R** [comment]
I think the -r should be added in  PSA addon settings tab.
